src/components/header/AccountMenu.tsx

```diff
@@ -1,106 +1,125 @@
-import React, { SFC } from 'react';
-import { ROUTE_PATHS, SUB_ROUTES } from '../../constants/routeNaming';
-import styled, { css } from 'styled-components';
-import { Link } from '../layout/StyledCopy';
-import { lighten } from 'polished';
-
-const menuList = [
-    {
-        path: '',
-        name: 'Account'
-    },
-    {
-        path: SUB_ROUTES.favorites,
-        name: 'Favorites'
-    },
-    {
-        path: SUB_ROUTES.dataSets,
-        name: 'My datasets'
-    },
-    {
-        path: SUB_ROUTES.pullRequests,
-        name: 'Pull requests'
-    }
-];
+import React, { PureComponent } from 'react';
+import styled from '../../styled-components';
+
+import { UserReducer } from '../../typings/store';
+import { MenuItemProp } from '../../typings';
+
+import { LOGIN_URL } from '../../constants/api';
+
+import { Subtitle } from '../layout/StyledCopy';
+import AvatarButton from './AvatarButton';
+import Tooltip, { ALIGN } from '../Tooltip';
+
+import { MENU_ITEMS } from '../../constants/global';
+import AccountMenuItem from './AccountMenuItem';
+import Avatar, { SIZE } from './Avatar';
 
 interface Props {
+    user: UserReducer;
     onLogOut: () => void;
-    isFooter?: boolean;
 }
 
-const AccountMenu: SFC<Props> = ({ onLogOut, isFooter }) => {
-
-    const MenuList = styled.ul` 
-        list-style: none;
-        margin: 0;
-        padding: 0;
-        background: ${props => isFooter ? 'transparent' : props.theme.colors.shade.medium};
-
-    ${
-        !isFooter
-        ?  `position: absolute;
-            border-radius: .25rem;
-            overflow: hidden;
-            width: 100%;
-            max-width: 20rem;
-            z-index: 2;
-            right: 1rem;`
-        : ''
+interface State {
+    isOpen: boolean;
+}
+
+const LoginButton = styled.a`
+    margin-top: 0;
+    padding: 0.25rem 0.5rem;
+    border: 1px solid ${props => props.theme.colors.white};
+    border-radius: .15rem;
+    font: ${props => props.theme.fonts.body};
+    color: ${props => props.theme.colors.white};
+    background-color: ${props => props.theme.colors.black};
+    
+    &:hover {
+        color: ${props => props.theme.colors.black};
+        background-color: ${props => props.theme.colors.shade.light};
     }
 `;
-    const MenuListItem = styled.li``;
 
-    const MenuLinkStyle = css`
-    display: inline-block;
+const AccountContainer = styled.div`
+    position: absolute;
+    top: 0;
+    right: 2rem;
+    height: 1.5rem;
+    bottom: 0;
+    margin: auto;
+`;
+
+const MenuHeader = styled.div`
     position: relative;
-    cursor: pointer;
-    width: 100%;
-    color: ${p => isFooter ? '#fff' : 'inherit'};
-    padding: ${p => isFooter ? '0' : '.5rem 1rem .5rem 3rem'};
+    padding: 1rem;
+`;
+
+const AvatarContainer = styled.figure`
+    position: absolute;
+    top: 0;
+    right: 0;
+`;
+
+const MenuList = styled.ul`
```

Comment:
???

Reply:
Kan default ul worden. Had er eerst styling in staan maar niet meer verwijderd.